### PR TITLE
Implement hover panel collection info header

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-6902

### CODE REVIEW

File: src/tiles/hover/tile-hover-pane.ts [hunk 1]
```diff
@@ -15,6 +22,10 @@ export class TileHoverPane extends LitElement {
 
   @property({ type: Boolean }) loggedIn: boolean = false;
 
+  @property({ type: Boolean }) mobileBreakpoint?: boolean;
+
+  @property({ type: Boolean }) currentWidth?: boolean;
```
[the remark this answers is not in the document]
Comment (nsharma123) [reply src/tiles/hover/tile-hover-pane.ts:27]: Mobile design: 
image

Desktop design:
image

File: src/tiles/hover/tile-hover-pane.ts [hunk 2]
```diff
@@ -15,6 +22,10 @@ export class TileHoverPane extends LitElement {
 
   @property({ type: Boolean }) loggedIn: boolean = false;
 
+  @property({ type: Boolean }) mobileBreakpoint?: boolean;
```
Comment (iisa) [inline src/tiles/hover/tile-hover-pane.ts:25]: q: are `currentWidth` & `mobileBreakpoint` numbers or a boolean?

File: src/tiles/hover/tile-hover-pane.ts [hunk 3]
```diff
@@ -23,25 +34,69 @@ export class TileHoverPane extends LitElement {
...
+          /><span>${collectionTitle}</span>
+        </a>
       </div>
     `;
   }
 
   static get styles(): CSSResultGroup {
+    const hoverPaneHeaderBGColor = css`var(--hoverPaneHeaderBGColor, #edf0ff)`;
+    const iaLinkColor = css`var(--ia-theme-link-color, #4b64ff)`;
+    const iaFontFamily = css`var(--ia-theme-base-font-family, "Helvetica Neue", Helvetica, Arial, sans-serif);`;
```
Comment (iisa) [inline src/tiles/hover/tile-hover-pane.ts:91]: q: wonder if we can just receive this implied, inherited from document?

File: test/tiles/hover/tile-hover-pane.test.ts [hunk 4]
```diff
@@ -16,4 +17,64 @@ describe('Tile Hover Pane', () => {
...
+    );
+
+    const container = el.shadowRoot?.querySelector('#container');
+    const listView = container?.querySelector('tile-list');
+    const headerElement = container?.querySelector('#list-line-header');
+
+    expect(container).to.exist;
+    expect(listView).to.exist;
+    expect(headerElement).not.exist;
+  });
```
Comment (iisa) [inline test/tiles/hover/tile-hover-pane.test.ts:79]: thanks for the tests! 👍

**Review by iisa — COMMENTED** [review]
a couple of questions.

**Comment by iisa** [comment]
image

image

File: src/tiles/hover/tile-hover-pane.ts (hunk 3 above)
Comment (nsharma123) [reply src/tiles/hover/tile-hover-pane.ts:91]: We are receiving these from document by just created variable here to group them at single place having fallback to insure they get correct values.

File: src/tiles/hover/tile-hover-pane.ts (hunk 2 above)
Comment (nsharma123) [reply src/tiles/hover/tile-hover-pane.ts:25]: These are should be numbers, sorry for typo (just copy pasted above property and forgot to change :grin:

**Pull request merged**